.travis.yml

```diff
@@ -4,5 +4,9 @@ python:
 
 # command to install dependencies
 install: pip install -r requirements.txt
+# enforce some code style standards
+before_script: 
+  - pip install flake8 --use-mirrors
+  - flake8 --ignore=E731,E226 .
```

Comment:
@pepol E226: 'missing whitespace around arithmetic operator', E731: 'do not assign a lambda expression, use a def'

Both are well intended but can arguably lead to both suboptimal code and sad faces which I'd like to avoid.